.github/workflows/testing.yml

```
@@ -139,6 +150,17 @@ jobs:
         with:
           name: rgbds-canary-win${{ matrix.bits }}
           path: bins
+      - name: Check test dependency repositories cache
+        id: test-deps-cache
+        uses: actions/cache@v3
+        with:
+          path: test
+          key: ${{ hashFiles('test/fetch-test-deps.sh') }}
+      - if: ${{ steps.test-deps-cache.outputs.cache-hit != 'true' }}
+        name: Fetch test dependency repositories
```

```suggestion
        name: Fetch test dependency repositories
        shell: bash
```
I think this is why the Windows tests are failing.

---

Ah yes, probably. I ignored the failure error cause I wasn't up to date with my fork and it was failing tests due to not being able to download zlib, didn't notice it's actually failing on something else. My bad, sorry!